UPGRADE-3.x.md

```diff
@@ -1,6 +1,28 @@
 UPGRADE 3.x
 ===========
 
+## Added constants for "show" and "list" templating types.
+
+You can use `TemplateRegistry` constants, like
+```
+$showMapper->add('foo', TemplateRegistry::TYPE_STRING)
+```
+instead of using directly a string value.
+```
+$showMapper->add('foo', 'string')
+```
+
+The list of available types can be found in [the documentation](docs/reference/field_types.rst).
+
+## Deprecated field types
```

Comment:
I'm sorry, I missed the line break in my previous suggestion.
```suggestion
## Deprecated field types

```

Reply:
No problem, i replace by `templating types` too and added the dot.

Reply:
I've removed the full stop on purpose, in order to be consistent with the previous sections in this document.

Reply:
Ok, it's removed